### PR TITLE
Adjusting application modules

### PR DESCRIPTION
**Pull request by mioyokohama**
Adjusted duplicate Plant Diseases modules so that only one remains with role restrictions.
This prevents users from being confused by duplicate modules.
The following commits are unnecessary, but have already been merged and are fine.
[A new plant, Cherry blossom (Sakura), has been added to the plant Table.](https://github.com/ServiceNowDevProgram/Plants/pull/454/commits/4d0e9c1d8703198380696ed945205a1012b841f5)

### CODE REVIEW

File: 9c697d609329ca10c7067a418bba1022/update/sys_metadata_link_419cb3bf839512108b1d10c5eeaad359.xml
Comment (wiz0floyd) [inline 9c697d609329ca10c7067a418bba1022/update/sys_metadata_link_419cb3bf839512108b1d10c5eeaad359.xml:1]: This file appears to be related to a separate PR. Please remove it from this PR or add a reference to it in your PR Description.

**Review by wiz0floyd — CHANGES_REQUESTED** [review]

**Comment by mioyokohama** [comment]
I've decided to close this PR and resubmit the request.
